docs/BUILD.md

```diff
@@ -1,84 +1,87 @@
-# Installation and Build Options
-## Prerequisites
+# Build Instructions
 
-|Build Type| Requirements|
-|:-----------------------|-------------------|
-|Use pre-built packages| Python 3.6, 3.7, or 3.8, TensorFlow v2.5.0|
-|Build from source| Python 3.6, 3.7, or 3.8, GCC 7.5 (Ubuntu 18.04, 20.04),  cmake 3.14 or higher, Bazelisk v1.7.5, virtualenv 16.0.0 or higher, patchelf 0.9, libusb 1.0.0|
+<!-- vscode-markdown-toc -->
+* 1. [Prerequisites](#Prerequisites)
```

Comment:
Extra * can be removed. This is adding extra bullet heading

Reply:
Auto generated using a vscode extensions, will manually modify it.